src/components/dev-hub/series.js

```diff
@@ -6,67 +6,50 @@ import { H3, P } from './text';
 import {
     colorMap,
     fontSize,
-    gradientMap,
     layer,
+    lineHeight,
     screenSize,
     size,
 } from './theme';
 
-const BULLET_BOX_WIDTH = '14px';
-const BULLET_SIZE = '22px';
+const BULLET_BOX_WIDTH = '30px';
+const BULLET_SIZE = 36;
+
+// Needed to allow more magenta to account for proper text coloring
+const BULLET_GRADIENT = `linear-gradient(
```

Comment:
Now that we have a lot of gradients everywhere, maybe we should create a helper function so that they can be created on the fly when we need them?

Reply:
This may or may not be a post-launch cleanup item though. What do you guys think?

Reply:
can we add these to the gradientMap in the theme file?

Reply:
We can, but I think they are only used in one place so that is why I didn't want to move them into theme. We can definitely parameterize though and I can work on that after this PR.